admin/class-acc_user_importer-admin.php

```diff
@@ -319,6 +322,19 @@ private function proccess_user_data ( $users ) {
 				continue;
 			}
 
+			switch($loginNameMapping) {
+				case 'ContactId':
+					$loginName = $userContactId;
+					break;
+				case 'firstname-lastname':
+					$loginName = "$userFirstName-$userLastName";
```

Comment:
I am not sure the above line makes sense.  Inside the quotes, the variables will not be expanded, right?
The way we had it in our system was 
$loginName = $userFirstName . " " . $userLastName;
Can we have that instead of with a dash?

Reply:
The variables are expanded if they are within [double-quotes](https://www.php.net/manual/en/language.types.string.php#language.types.string.parsing). I did misremember the ID and thought you used a dash, I'll change it to a space.